docs/kit/logistics/transport/couriers.md

```diff
@@ -19,3 +15,33 @@ Kit data for couriers:
 In all circumstances a shipment of a kit must have at least £500 worth of insurance.
 
 Note that the above package details are repeated in the loan extension instructions in [return-shipping-pack.git](https://github.com/srobo/return-shipping-pack/tree/master/instructions) and therefore should be updated there when the details change.
+
+## [DPD Local](https://www.dpdlocal.co.uk/)
+
+We have used Interlink [http://www.interlinkdirect.co.uk/](http://www.interlinkdirect.co.uk/) in the past to ship kits. Interlink were bought by DPD and rebranded to "DPD Local" in 2017.
+
+We were unable to send LiPo batteries via DPD Local when preparing for SR2022, as the majority of our batteries (any batteries not manufactured by Overlander) do not have the appropriate certifications.
+
+## [UPS](https://www.ups.com/gb/en/Home.page)
+
+UPS are willing to ship our kits, including LiPos. However, this is very expensive when using the consumer services.
+
+For SR2022, we shipped kits via the UPS Business account of a friendly company, herein referred to as S5-UPS.
+
+### Additional Packing Requirements for UPS
+
+* Lithium Polymer batteries must be individually packaged and sealed
+  * This is to prevent shorts.
+  * The batteries do not need to be in a LiPo bag, although we would strongly recommend that they are.
+* A LiPo warning sticker must be affixed to the outside of the box.
+* The LiPos must be secured such that they will not move during shipping.
+* The box must be flat-sided and "square" to qualify for low shipping rates
+
+### Additional Packing Requirements for S5-UPS
+
+* LiPos must be individually inside jiffy bags, then inside the LiPo bag.
```

Comment:
The below packaging info mentions that batteries "must" be in their own jiffy bags, however that being in this section here implies it's supplier (S5) specific? True or not, I wonder if we should just mandate it anyway?